.github/scripts/detect_unchanged_ids.py

```diff
@@ -0,0 +1,116 @@
+import os
+import sys
+import argparse
+
+try:
+    from compare_locales import parser
+except ImportError as e:
+    print("FATAL: make sure that dependencies are installed")
+    print(e)
+    sys.exit(1)
+
+
+def get_file_list(path, reference_locale="locale/en"):
```

Comment:
The goal would be to be able to reuse this in other places, so it needs to be a parameter that you pass to the script, and also an actual locale code (`en` in this case)

Reply:
On second thoughts, I see how using a locale code might make things harder, so we can pass the "path to reference locale" as argument.